### PR TITLE
chore: update forma to v4

### PR DESCRIPTION
**Pull request by KrisQ**
## Why are these changes introduced?

Addresses [ENG-9419](https://nacelle.atlassian.net/browse/ENG-9419) 

## What is this pull request doing?

Updates forma to v4

## How to test

- Make sure node 16 is being used
- `npm run start` 
- Choose the Prairie wind apparel space with the `staging` environment and the app should be installed by default

## Checklist

- [x] You can follow your own "How to Test" instructions and get the expected result.

[ENG-9419]: https://nacelle.atlassian.net/browse/ENG-9419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Dialog.tsx [hunk 1]
```diff
@@ -199,6 +201,7 @@ export default class Dialog extends Component<DialogProps, DialogState> {
   }
 
   setSelectedTab = (id: string) => {
+    console.log('setSelectedTab', id)
```
Comment (NWRichmond) [inline src/components/Dialog.tsx:204]: Was this here for debugging purposes?

```suggestion
```

File: src/components/NacelleReferences/NacelleReferences.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ const NacelleReferences = (props: NacelleReferencesProps) => {
     getNacelleFields(cmaAdapter, appId, spaceId, envId)
       .then((res) => setData(res))
       .catch((err) => {
-        throw new Error(err)
+        throw new Error(JSON.stringify(err))
```
Comment (NWRichmond) [inline src/components/NacelleReferences/NacelleReferences.tsx:24]: Is `err` not a string?

**Review by NWRichmond — DISMISSED** [review]
Everything is working as expected when running locally on Node.js `16.20.0` with the Prairie Wind Apparel space (`Experimental` environment). Nice!

There are just a couple of areas where I have questions. But nothing major or worthy of a re-review.

Thanks for this PR 🤝

**Comment by KrisQ** [comment]
Thanks @NWRichmond, all the superfluous is gone!

**Review by NWRichmond — APPROVED** [review]
Sweeeeeet!

**Pull request merged**